docs/administration/authentication-providers/okta-authentication.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Okta authentication
+description: Octopus Deploy can use Okta authentication to identify users.
+position: 6
+---
+
+:::hint
+Okta authentication is available in Octopus Deploy 3.16 and later
+:::
+
+To use Okta authentication with Octopus you will need to:
+
+1. Configure Okta to trust your Octopus Deploy instance (by setting it up as an App in Okta)
+2. Configure your Octopus Deploy instance to trust and use Okta as an Identity Provider.
+
+## Configuring Okta {#Oktaauthentication-ConfiguringOkta}
+
+The first steps are to configure Okta to trust your instance of Octopus Deploy by configuring an App in your Okta account.
+
+### Set up an App {#Oktaauthentication-SetupanApp}
+
+You must first have an account at [Okta](https://www.okta.com/). You can sign up for a free [developer account](https://developer.okta.com/signup/).
+
+Once you have an account, log in to the Okta admin portal.
+
+:::hint
+After signing up to Okta you will receive your own url to access the Okta portal. For a developer account, it will look something similar to: `https://dev-xxxxxx-admin.oktapreview.com`
+:::
+
+1. Select the Applications tab and click the **Add Application** button
+
+   ![](/docs/images/okta-authentication/okta-add-app.png "width=500")
+
+2. Click the **Create New App** button
+
+   ![](/docs/images/okta-authentication/okta-create-new-app.png "width=500")
+
+3. Choose **Web** for the **Platform** and **OpenID Connect** for the **Sign on method** and click the **Create** button.
+
+   ![](/docs/images/okta-authentication/okta-new-app-integration.png "width=400")
+
+4. Enter an **Application Name** like Octopus Deploy and for the **Login redirect URIs** enter `https://octopus.example.com/api/users/authenticatedToken/Okta` replacing `https://octopus.example.com` with the public url of your Octopus server, and click the **Save** button.
+
+   ![](/docs/images/okta-authentication/okta-create-openid-integration.png "width=500")
+
+:::hint
+**Reply URLs are case-sensitive**
+Please take care when adding this URL! They are **case-sensitive** and can be sensitive to trailing **slash** characters
+:::
+
+:::hint
+**Not using SSL?**
+That's OK, you can use `http` if you do not have SSL enabled on your Octopus Server. Please beware of the security implications in accepting a security token over an insecure channel.
```

Comment:
Not enthusiastic about "That's OK". We should really be pushing them down the secure route. 

Maybe something like:
> We highly recommend using SSL, but we know its not always possible (it might be [easier than you think](https://octopus.com/docs/administration/lets-encrypt-integration)). If you absolutely have to, please beware of the security implications in accepting a security token over an insecure channel.